assets/models/system/Mistral-7B-v0-1/spec.yaml

```diff
@@ -4,11 +4,11 @@ path: ./
 properties:
   SharedComputeCapacityEnabled: true
   SHA: f966a600a1f2c35a067dcb16b24bbf9ac3b85799
-  inference-min-sku-spec: 12|2|224|672
+  inference-min-sku-spec: 12|1|220|64
```

Comment:
lets check if model size would allow to be deployed on this spec? Please add deployment valdn for this one

Reply:
Model size is ~30GB . Disk space should not be a problem